src/sparsevector.jl

```diff
@@ -145,10 +157,36 @@ function nonzeroinds(x::SparseColumnView)
 end
 nonzeroinds(x::SparseVectorView) = nonzeroinds(parent(x))
 
+# return the first and last nonzero indices of the parent that belong to the view
+# return end+1:end if no nonzero in the parent
+function _partialview_end_indices(x::SparseVectorPartialView)
+    p = parent(x)
+    nzinds = nonzeroinds(p)
+    if isempty(nzinds)
+        first_idx = nzinds[begin]
```

Comment:
I'm confused, shouldn't this be out of bounds? It's empty.

Reply:
good catch, I added a test to cover the case too